common/app/views/fragments/amp/header.scala.html

```diff
@@ -16,13 +16,17 @@ <h1 class="u-h">The Guardian</h1>
         </a>
 
         <nav class="header__nav">
+            <ul class="header__nav__primary-links">
             @NewNavigation.PrimaryLinks.map { link =>
```

Comment:
Nit: the spacing is off here. make sure the opening and closing `ul` tags match!